typings/lib/httpPipelineLogLevel.d.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * The different levels of logs that can be used with the HttpPipelineLogger.
+ */
+export declare enum HttpPipelineLogLevel {
+    /**
+     * A log level that indicates that no logs will be logged.
+     */
+    OFF = 0,
```

Comment:
Already mentioned in another comment, but consider just assigning string values instead of numeric values here.

Reply:
I'll have to make sure that the comparison in RequestPolicyOptions.shouldLog() still works after changing this to a string enum.

Reply:
Changing this to a string enum did break the comparison that I need. I found [this trick](https://stackoverflow.com/a/42172322) though, and it works great.

Reply:
Oh, this is about the enum values having an associated severity, right?

Reply:
Correct.

Reply:
I did change HttpMethod to a string enum, though, because they don't need to be compared relative to each other.